Application/DVector.pde

```diff
@@ -0,0 +1,28 @@
+class DVector {
```

Comment:
maybe add default constructor (without parameter)? it will be useful to prevent magic numbers